xml/hardware_nvdimm.xml

```diff
@@ -825,6 +825,254 @@ I/O size (minimum/optimal): 4096 bytes / 4096 bytes</screen>
+    <filename>/dev/pmem1</filename> or <filename>/dev/dax0</filename>.
+   </para>
+   <para>
+    For example, imagine a single region made up of space from three NVDIMMs
+    which has been configured as three namespaces, as follows: 
+   </para>
+   <informaltable>
+    <tgroup cols="4">
+    <colspec colname="col1"/>
+    <colspec colname="col2"/>
+    <colspec colname="col3"/>
+    <colspec colname="col4"/>
+     
+     <tbody>
+     <row>
+      <entry>
+       <para>
+        NVDIMM 0
+       </para>
+      </entry>
+      <entry>
+       <para>
+        region0
+       </para>
+      </entry>
+      <entry>
+       <para>
+        /dev/pmem1
+       </para>
+      </entry>
+      <entry></entry>
+     </row>
+     
+     <row>
+      <entry>
+       <para>
+        NVDIMM 1
+       </para>
+      </entry>
+      <entry>
+       <para>
+        <emphasis>[X]</emphasis>
+       </para>
+      </entry>
+      <entry>
+       <para>
+        /dev/pmem2s
+       </para>
+      </entry>
+      <entry></entry>
+     </row>
+     
+     <row>
+      <entry>
+       <para>
+        NVDIMM 2
+       </para>
+      </entry>
+      <entry>
+       <para>
+        /dev/dax0
+       </para>
+      </entry>
+      <entry namest="col3" nameend="col4"></entry>
+     </row>
+     
+    </tbody>
+    </tgroup>
+   </informaltable>
+   
+   <para>
+    In our example, the part of <filename>region0</filename> labelled as
+    <emphasis>[X]</emphasis> has been damaged or become defective.
+   </para>
+   <para>
+    You must:
+   </para>
+   <orderedlist>
+    <listitem>
+     <para>
+      Identify which NVDIMM module(s) contain the affected region.
+     </para>
+     <para>
+      This is particularly important if the region is interleaved across more
+      than one NVDIMM.
+     </para>
+    </listitem>
+    
+    <listitem>
+     <para>
+      Back up the contents of any other namespaces on the affected NVDIMM.
+     </para>
+     <para>
+      In this example, you must back up the contents of <filename>/dev/pmem2s</filename>.
+     </para>
+    </listitem>
+    
+    <listitem>
+     <para>
+      Identify the relationship between the namespaces and the physical position
+      of the NVDIMM (in which motherboard memory slot it is located).
+     </para>
+     <para>
+      The server must be shut down, its cover
+      removed, and the defective modules found, removed and replaced.
+     </para>
+    </listitem>
+   </orderedlist>
+  </sect2>
+  <sect2 xml:id="sec-nvdimm-testing">
+   <title>Testing Persistent Memory</title>
+   <note>
+    <title>Prerequisites for Fault-Finding</title>
+    <para>
+     For testing, the <filename>nfit_test</filename> kernel module is required.
+    </para>
+   </note> 
+   <para>
+    The testing procedure is described in detail on the Github page for the
+    <command>ndctl</command> command, in steps 1&mdash;4 of the section
+    <literal>Unit test</literal>. A link can be found under "For More
```

Comment:
If you meant to refer to the 'For More Information' section in the same document, use an xref: 

          [...] in <xref linkend="sec-nvdimm-moreinfo"/>.

In addition, why not using active voice here? Find a link in [....]